tests/control/ControllerTest.php

```diff
@@ -29,6 +29,7 @@ class ControllerTest extends FunctionalTest {
 
 	public function setUp() {
 		parent::setUp();
+        Config::inst()->update('SilverStripe\\Control\\Director', 'alternate_base_url', '/');
```

Comment:
I'm loathed to block merging due to whitespace issues, but this is spaced and the rest of the file is tabbed